Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml

```diff
@@ -0,0 +1,35 @@
+- type: job
+  id: Paramedic
+  name: job-name-paramedic
+  description: job-description-paramedic
+  playTimeTracker: JobParamedic
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 1800
+  startingGear: ParamedicGear
+  icon: "Paramedic"
+  supervisors: job-supervisors-cmo
+  access:
+  - Medical
+  - Maintenance
+  - External
+  - Service
+  extendedAccess:
+  - Chemistry
```

Comment:
Don't think chem access is needed here-

Reply:
I put low pop access on paramedic because medical doctor has it.